src/NzbDrone.Core/Notifications/Notifiarr/NotifiarrProxy.cs

```diff
@@ -84,10 +93,19 @@ private void ProcessNotification(StringDictionary message, NotifiarrSettings set
             }
             catch (HttpException ex)
             {
-                if (ex.Response.StatusCode == HttpStatusCode.BadRequest)
+                switch (true)
```

Comment:
Should it just switch on the status code? Switching on true looks really weird...

Reply:
It cant in this situation because CF are not in the ENUM

Reply:
Could you switch on `(int)ex.Response.StatusCode`?

Reply:
FWIW i think this is temporary anyways as i am going to look at fixing the global CF interceptor. I'll need to update this again when it switches to the API and at that point will likely remove all but the 400 & 401 so it can then `switch` or `if`

Reply:
> global CF interceptor

There's some response handling rework on Prowlarr `resp-http` that may have some useful pieces to pick when that can of worms gets opened

Reply:
Probably a bit easier to read switching on the int casted code instead of true like ta mentioned. I agree on that if it's not too much trouble.